rotkehlchen/chain/ethereum/modules/yearn/utils.py

```diff
@@ -134,11 +134,11 @@ def query_yearn_vaults(db: 'DBHandler') -> None:
     with GlobalDBHandler().conn.write_ctx() as write_cursor:
         # Delete the old value cached and store in the cache the amount of vaults
         # processed in this response.
-        globaldb_delete_general_cache(
+        globaldb_delete_cache(
```

Comment:
This here seems to also be a kind of cache that should be in the unique cache no? What does it represent? It seems to represent the number of yearn vaults stored? Or am I reading the code wrong? In which case the unique key/value cache should be used and instead of a delete/set an insert or replace should happen here

Reply:
will look at it and determine